2019-11-04 15:54:55,663 - __main__ - ERROR - Error in DB - name 'logger' is not defined
2019-11-04 15:54:55,664 - __main__ - INFO - Finished inserting records.
2019-11-04 15:58:14,939 - __main__ - ERROR - Error in connection - FATAL:  password authentication failed for user "postgres"

2019-11-04 15:58:14,940 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 15:58:14,940 - __main__ - INFO - Finished inserting records.
2019-11-04 15:58:38,289 - __main__ - ERROR - Error in connection - FATAL:  password authentication failed for user "postgres"

2019-11-04 15:58:38,290 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 15:58:38,290 - __main__ - INFO - Finished inserting records.
2019-11-04 15:59:29,616 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 15:59:29,616 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 15:59:29,617 - __main__ - INFO - Finished inserting records.
2019-11-04 16:01:24,017 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:01:24,018 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:01:24,018 - __main__ - INFO - Finished inserting records.
2019-11-04 16:06:57,995 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:06:57,995 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:06:57,996 - __main__ - INFO - Finished inserting records.
2019-11-04 16:08:25,227 - __main__ - ERROR - Error in connection - FATAL:  password authentication failed for user "postgres"

2019-11-04 16:08:25,227 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:08:25,228 - __main__ - INFO - Finished inserting records.
2019-11-04 16:10:32,718 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:10:32,719 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:10:32,719 - __main__ - INFO - Finished inserting records.
2019-11-04 16:10:50,636 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:10:50,637 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:10:50,638 - __main__ - INFO - Finished inserting records.
2019-11-04 16:11:40,584 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:11:40,586 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:11:40,586 - __main__ - INFO - Finished inserting records.
2019-11-04 16:12:01,672 - __main__ - ERROR - Error in connection - FATAL:  database "test_db" does not exist

2019-11-04 16:12:01,673 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:12:01,673 - __main__ - INFO - Finished inserting records.
2019-11-04 16:12:37,890 - __main__ - INFO - Finished inserting records.
2019-11-04 16:22:36,021 - __main__ - ERROR - Error in DB - module 'create_db_table' has no attribute 'table'
2019-11-04 16:22:36,021 - __main__ - INFO - Finished inserting records.
2019-11-04 16:25:20,176 - __main__ - INFO - Finished inserting records.
2019-11-04 16:41:11,982 - __main__ - ERROR - Error in connection - FATAL:  password authentication failed for user "postgres"

2019-11-04 16:41:11,983 - __main__ - ERROR - 'NoneType' object has no attribute 'cursor'
2019-11-04 16:41:11,984 - __main__ - INFO - Finished inserting records.
2019-11-04 16:41:34,710 - __main__ - INFO - Finished inserting records.
2019-11-04 16:42:18,298 - __main__ - ERROR - Unable to insert record - null value in column "datetime" violates not-null constraint
DETAIL:  Failing row contains (1, null, ., config.ini, 118, 75e2f21c692cb4fe6601f3c417b72350, 8aeced47bad30e683a0e071fe4edc7ba48ae9b9a).

2019-11-04 16:42:18,299 - __main__ - INFO - Finished inserting records.
2019-11-04 16:43:29,740 - __main__ - INFO - Finished inserting records.
2019-11-04 16:45:43,923 - __main__ - INFO - Finished inserting records.
2019-11-04 16:48:30,268 - __main__ - INFO - Finished inserting records.
2019-11-04 17:02:31,538 - __main__ - INFO - Naming default_file
2019-11-04 17:02:31,539 - __main__ - INFO - Result: default_file_20191104_170231
2019-11-04 17:02:31,539 - __main__ - INFO - Opening new .csv file.
2019-11-04 17:02:31,539 - __main__ - INFO - Including the ff. headers: ['Parent Path', 'File Name', 'File Size', 'MD5', 'SHA1'].
2019-11-04 17:02:31,540 - __main__ - INFO - Writing rows...
2019-11-04 17:02:31,552 - __main__ - INFO - Finished with .csv file. Onto zipping...
2019-11-04 17:02:31,552 - __main__ - INFO - Zipping default_file_20191104_170231.csv
2019-11-04 17:02:31,553 - __main__ - INFO - Creating zip...
2019-11-04 17:02:31,557 - __main__ - INFO - Zipping done.
2019-11-04 17:02:31,558 - __main__ - INFO - Finished creating default_file_20191104_170231.csv.zip.
2019-11-04 17:04:31,704 - __main__ - INFO - Naming default_file
2019-11-04 17:04:31,705 - __main__ - INFO - Result: default_file_20191104_170431
2019-11-04 17:04:31,705 - __main__ - INFO - Opening new .csv file.
2019-11-04 17:04:31,706 - __main__ - INFO - Including the ff. headers: ['Parent Path', 'File Name', 'File Size', 'MD5', 'SHA1'].
2019-11-04 17:04:31,706 - __main__ - INFO - Writing rows...
2019-11-04 17:04:31,722 - __main__ - INFO - Finished with .csv file. Onto zipping...
2019-11-04 17:04:31,722 - __main__ - INFO - Zipping default_file_20191104_170431.csv
2019-11-04 17:04:31,723 - __main__ - INFO - Creating zip...
2019-11-04 17:04:31,727 - __main__ - INFO - Zipping done.
2019-11-04 17:04:31,728 - __main__ - INFO - Finished creating default_file_20191104_170431.csv.zip.
2019-11-04 17:04:45,116 - __main__ - INFO - Naming default_file
2019-11-04 17:04:45,117 - __main__ - INFO - Result: default_file_20191104_170445
2019-11-04 17:04:45,117 - __main__ - INFO - Opening new .csv file.
2019-11-04 17:04:45,118 - __main__ - INFO - Including the ff. headers: ['Parent Path', 'File Name', 'File Size', 'MD5', 'SHA1'].
2019-11-04 17:04:45,118 - __main__ - INFO - Writing rows...
2019-11-04 17:04:45,134 - __main__ - INFO - Finished with .csv file. Onto zipping...
2019-11-04 17:04:45,134 - __main__ - INFO - Zipping default_file_20191104_170445.csv
2019-11-04 17:04:45,135 - __main__ - INFO - Creating zip...
2019-11-04 17:04:45,141 - __main__ - INFO - Zipping done.
2019-11-04 17:04:45,141 - __main__ - INFO - Finished creating default_file_20191104_170445.csv.zip.
2019-11-04 17:06:53,344 - __main__ - INFO - Finished inserting records.
2019-11-04 17:07:28,858 - __main__ - INFO - Finished inserting records.
2019-11-04 17:08:55,801 - __main__ - INFO - Naming default_file
2019-11-04 17:08:55,802 - __main__ - INFO - Result: default_file_20191104_170855
2019-11-04 17:08:55,802 - __main__ - INFO - Opening new .csv file.
2019-11-04 17:08:55,802 - __main__ - INFO - Including the ff. headers: ['Parent Path', 'File Name', 'File Size', 'MD5', 'SHA1'].
2019-11-04 17:08:55,803 - __main__ - INFO - Writing rows...
2019-11-04 17:08:55,815 - __main__ - INFO - Finished with .csv file. Onto zipping...
2019-11-04 17:08:55,815 - __main__ - INFO - Zipping default_file_20191104_170855.csv
2019-11-04 17:08:55,816 - __main__ - INFO - Creating zip...
2019-11-04 17:08:55,822 - __main__ - INFO - Zipping done.
2019-11-04 17:08:55,822 - __main__ - INFO - Finished creating default_file_20191104_170855.csv.zip.
2019-11-04 17:31:42,256 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:31:42,263 - __main__ - INFO - Finished inserting records.
2019-11-04 17:34:27,947 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:34:27,947 - __main__ - INFO - Finished inserting records.
2019-11-04 17:35:21,024 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:35:21,025 - __main__ - INFO - Finished inserting records.
2019-11-04 17:38:38,569 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:38:38,570 - __main__ - INFO - Finished inserting records.
2019-11-04 17:41:45,858 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:41:45,858 - __main__ - INFO - Finished inserting records.
2019-11-04 17:45:00,604 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:45:00,605 - __main__ - INFO - Finished inserting records.
2019-11-04 17:45:40,313 - __main__ - ERROR - Error in DB - name 'create_db_table' is not defined
2019-11-04 17:45:40,313 - __main__ - INFO - Finished inserting records.
2019-11-04 17:48:56,118 - __main__ - INFO - Finished inserting records.
2019-11-04 17:49:35,365 - __main__ - INFO - Finished inserting records.
2019-11-04 17:52:54,293 - __main__ - INFO - Finished inserting records.
2019-11-12 15:46:29,199 - __main__ - INFO - Naming default_file
2019-11-12 15:46:29,208 - __main__ - INFO - Result: default_file_20191112_154629
2019-11-12 15:46:29,209 - __main__ - INFO - Opening new .json file.
2019-11-12 15:46:29,210 - __main__ - INFO - Writing rows...
2019-11-12 15:46:29,335 - __main__ - INFO - Finished with .json file. Onto zipping...
2019-11-12 15:46:29,335 - __main__ - INFO - Zipping default_file_20191112_154629.json
2019-11-12 15:46:29,336 - __main__ - INFO - Creating zip...
2019-11-12 15:46:29,351 - __main__ - INFO - Zipping done.
2019-11-12 15:46:29,351 - __main__ - INFO - Finished creating default_file_20191112_154629.json.zip.
2019-11-12 16:42:03,721 - __main__ - INFO - Naming default_file
2019-11-12 16:42:03,722 - __main__ - INFO - Result: default_file_20191112_164203
2019-11-12 16:42:03,733 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-12 16:42:03,734 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=140, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 52077, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-12 16:42:03,735 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001A72E23A408>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001A72E23A408> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-12 16:42:03,747 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001A72E23A408> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:42:03,748 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001A72E23A408> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:42:03,748 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001A72E23A408> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:42:03,749 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-12 16:54:48,091 - __main__ - INFO - Naming default_file
2019-11-12 16:54:48,092 - __main__ - INFO - Result: default_file_20191112_165448
2019-11-12 16:54:48,092 - __main__ - ERROR - Unable to send to queue!
2019-11-12 16:56:32,451 - __main__ - INFO - Naming default_file
2019-11-12 16:56:32,451 - __main__ - INFO - Result: default_file_20191112_165632
2019-11-12 16:56:32,462 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-12 16:56:32,463 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=836, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 52126, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-12 16:56:32,465 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000022475E786C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000022475E786C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-12 16:56:32,469 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000022475E786C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:56:32,470 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000022475E786C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:56:32,471 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000022475E786C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 16:56:32,472 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-12 17:03:13,788 - __main__ - INFO - Naming default_file
2019-11-12 17:03:13,788 - __main__ - INFO - Result: default_file_20191112_170313
2019-11-12 17:03:13,800 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-12 17:03:13,801 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=768, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 52152, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-12 17:03:13,802 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001823FE07748>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001823FE07748> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-12 17:03:13,807 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001823FE07748> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:03:13,808 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001823FE07748> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:03:13,809 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001823FE07748> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:03:13,810 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-12 17:07:31,273 - __main__ - INFO - Naming default_file
2019-11-12 17:07:31,274 - __main__ - INFO - Result: default_file_20191112_170731
2019-11-12 17:07:31,284 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-12 17:07:31,285 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=768, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 52181, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-12 17:07:31,287 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000139BCD29708>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000139BCD29708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-12 17:07:31,293 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000139BCD29708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:07:31,293 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000139BCD29708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:07:31,294 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000139BCD29708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-12 17:07:31,294 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 08:30:28,702 - __main__ - INFO - Naming default_file
2019-11-13 08:30:28,715 - __main__ - INFO - Result: default_file_20191113_083028
2019-11-13 08:30:28,727 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 08:30:28,727 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=720, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 52863, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 08:30:28,729 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000017249A785C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000017249A785C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 08:30:28,734 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000017249A785C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 08:30:28,734 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000017249A785C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 08:30:28,735 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000017249A785C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 08:30:28,735 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 11:19:07,309 - __main__ - INFO - Naming default_file
2019-11-13 11:19:07,310 - __main__ - INFO - Result: default_file_20191113_111907
2019-11-13 11:19:07,324 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 11:19:07,325 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=880, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54556, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:07,326 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 11:19:07,332 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:07,333 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:07,333 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:07,334 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 11:19:07,405 - pika.channel - WARNING - Received remote Channel.Close (404): "NOT_FOUND - no exchange 'logs' in vhost '/'" on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001E2353A96C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
2019-11-13 11:19:07,406 - __main__ - ERROR - Unable to json.
2019-11-13 11:19:07,406 - pika.adapters.blocking_connection - INFO - Closing connection (200): Normal shutdown
2019-11-13 11:19:07,407 - pika.connection - INFO - Closing connection (200): 'Normal shutdown'
2019-11-13 11:19:07,408 - pika.adapters.utils.io_services_utils - INFO - Aborting transport connection: state=1; <socket.socket fd=880, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54556, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:07,408 - pika.adapters.utils.io_services_utils - INFO - _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=880, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54556, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:07,408 - pika.adapters.utils.io_services_utils - INFO - Deactivating transport: state=1; <socket.socket fd=880, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54556, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:07,409 - pika.connection - INFO - AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
2019-11-13 11:19:07,409 - pika.connection - INFO - Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
2019-11-13 11:19:07,409 - pika.adapters.utils.io_services_utils - INFO - Closing transport socket and unlinking: state=3; <socket.socket fd=880, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54556, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:07,410 - pika.adapters.blocking_connection - INFO - User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
2019-11-13 11:19:57,546 - __main__ - INFO - Naming default_file
2019-11-13 11:19:57,546 - __main__ - INFO - Result: default_file_20191113_111957
2019-11-13 11:19:57,555 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 11:19:57,556 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=828, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54568, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 11:19:57,557 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000264BABC9708>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000264BABC9708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 11:19:57,561 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000264BABC9708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:57,561 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000264BABC9708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:57,561 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000264BABC9708> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 11:19:57,563 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:13:04,894 - __main__ - INFO - Naming default_file
2019-11-13 12:13:04,895 - __main__ - INFO - Result: default_file_20191113_121304
2019-11-13 12:13:11,854 - __main__ - INFO - Naming default_file
2019-11-13 12:13:11,855 - __main__ - INFO - Result: default_file_20191113_121311
2019-11-13 12:13:11,864 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 12:13:11,865 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=692, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54690, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 12:13:11,866 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000024234B78688>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000024234B78688> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 12:13:11,870 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000024234B78688> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:13:11,870 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000024234B78688> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:13:11,870 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000024234B78688> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:13:11,871 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:16:37,659 - __main__ - INFO - Naming default_file
2019-11-13 12:16:37,659 - __main__ - INFO - Result: default_file_20191113_121637
2019-11-13 12:16:37,668 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 12:16:37,669 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=760, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54714, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 12:16:37,670 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C1491A93C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C1491A93C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 12:16:37,674 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C1491A93C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:16:37,675 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C1491A93C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:16:37,675 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C1491A93C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:16:37,675 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:17:55,252 - __main__ - INFO - Naming default_file
2019-11-13 12:17:55,252 - __main__ - INFO - Result: default_file_20191113_121755
2019-11-13 12:17:55,263 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 12:17:55,264 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=716, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54731, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 12:17:55,265 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000002304F0E7388>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000002304F0E7388> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 12:17:55,270 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000002304F0E7388> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:17:55,271 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000002304F0E7388> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:17:55,271 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000002304F0E7388> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:17:55,288 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:18:38,496 - __main__ - INFO - Naming default_file
2019-11-13 12:18:38,496 - __main__ - INFO - Result: default_file_20191113_121838
2019-11-13 12:18:38,505 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 12:18:38,506 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=504, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54737, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 12:18:38,507 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C9E18A76C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C9E18A76C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 12:18:38,511 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C9E18A76C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:18:38,511 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C9E18A76C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:18:38,513 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001C9E18A76C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:18:38,514 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:23:46,814 - __main__ - INFO - Naming default_file
2019-11-13 12:23:46,814 - __main__ - INFO - Result: default_file_20191113_122346
2019-11-13 12:23:46,823 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 12:23:46,824 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=764, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54786, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 12:23:46,825 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001CED0F18648>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001CED0F18648> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 12:23:46,829 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001CED0F18648> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:23:46,830 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001CED0F18648> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:23:46,830 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x000001CED0F18648> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 12:23:46,831 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 12:23:46,833 - root - INFO - [x] Sent {"config.ini": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\config.ini", "File Name": "config.ini", "File Size": 228, "MD5": "622f0b5a1491e1233a423ba88de4c9e7", "SHA1": "f87d280e08ea6fa43156a548b24434d0e0f40993"}]} to queue!
2019-11-13 12:23:46,841 - root - INFO - [x] Sent {"create_db_table.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\create_db_table.py", "File Name": "create_db_table.py", "File Size": 2180, "MD5": "85242881acb57bf4586b92b25a33ce87", "SHA1": "fa29674302cdafa0b2dfdcc1065706ab00714770"}]} to queue!
2019-11-13 12:23:46,842 - root - INFO - [x] Sent {"listdir.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir.py", "File Name": "listdir.py", "File Size": 9449, "MD5": "a94142dc823f3ce5b9173f3be417795b", "SHA1": "43ad07a5fb17c63e8446f3265a91524adad4c578"}]} to queue!
2019-11-13 12:23:46,857 - root - INFO - [x] Sent {"listdir_error.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_error.log", "File Name": "listdir_error.log", "File Size": 3581, "MD5": "865a8e5ecbdd544388eb606891ac7fee", "SHA1": "89c685ce17bebad2cfb675e84331febdad89d0c8"}]} to queue!
2019-11-13 12:23:46,866 - root - INFO - [x] Sent {"listdir_info.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_info.log", "File Name": "listdir_info.log", "File Size": 37690, "MD5": "98aec271b91efa879efe7b1b5a970650", "SHA1": "54ec96a997bae3a8407ca3e0e7e3eb0cdf7e8106"}]} to queue!
2019-11-13 12:23:46,867 - root - INFO - [x] Sent {"logging_listdir.yaml": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\logging_listdir.yaml", "File Name": "logging_listdir.yaml", "File Size": 861, "MD5": "7ee106c8c2dc81c0cd98ad21d4b2481f", "SHA1": "984571cf9fb34a7c9747a9d2dbd36bd0bbb7549c"}]} to queue!
2019-11-13 12:23:46,872 - root - INFO - [x] Sent {"MANIFEST.in": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\MANIFEST.in", "File Name": "MANIFEST.in", "File Size": 26, "MD5": "0d46f74b56ba4d6c16fe2ac11d437a32", "SHA1": "e0613788cd5822eb7a958bb0ffdfed522c9ef43b"}]} to queue!
2019-11-13 12:23:46,894 - root - INFO - [x] Sent {"sample.csv.csv": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\sample.csv.csv", "File Name": "sample.csv.csv", "File Size": 3922, "MD5": "3cc623eed556fd4dd8c3922b209f89c9", "SHA1": "ebc128aa0b2ef88228711b9bb2a42bdf76d60071"}]} to queue!
2019-11-13 12:23:46,907 - root - INFO - [x] Sent {"test.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\test.py", "File Name": "test.py", "File Size": 608, "MD5": "f0e8fe7e46fb2b4ec0a571cb689f2437", "SHA1": "21fc9ad28a03f8636c37e7a818bb62f88230092a"}]} to queue!
2019-11-13 12:23:46,909 - root - INFO - [x] Sent {"__init__.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__init__.py", "File Name": "__init__.py", "File Size": 280, "MD5": "6036a582623e1da0890729d34c84874b", "SHA1": "92f8a64d57c926c6d37e487f5ea6a2d979b03f7d"}]} to queue!
2019-11-13 12:23:46,912 - root - INFO - [x] Sent {"create_db_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_db_table.cpython-37.pyc", "File Name": "create_db_table.cpython-37.pyc", "File Size": 2196, "MD5": "4b5c10918601efdbf85c9eaf4a3fb9a8", "SHA1": "801d646d90f2795cf47db3ede1b718b5f256aa0d"}]} to queue!
2019-11-13 12:23:46,913 - root - INFO - [x] Sent {"create_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_table.cpython-37.pyc", "File Name": "create_table.cpython-37.pyc", "File Size": 939, "MD5": "0f0659634a6217b8273fd46dd2c22e24", "SHA1": "e2eb26a536a703736c6236fab21de58c09250106"}]} to queue!
2019-11-13 12:23:46,915 - root - INFO - [x] Sent {"listdir.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\listdir.cpython-37.pyc", "File Name": "listdir.cpython-37.pyc", "File Size": 7662, "MD5": "e9df279f9bba98f87293778f30db644e", "SHA1": "b358cd5178093a81931932e172e023c56664c401"}]} to queue!
2019-11-13 12:23:46,917 - root - INFO - [x] Sent {"__init__.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\__init__.cpython-37.pyc", "File Name": "__init__.cpython-37.pyc", "File Size": 366, "MD5": "9b2bc2af4c7f78c59c397c0040006c82", "SHA1": "f026e605e13aa3eb3d17edeae9bfee768bacb081"}]} to queue!
2019-11-13 13:05:22,693 - __main__ - INFO - Naming default_file
2019-11-13 13:05:22,693 - __main__ - INFO - Result: default_file_20191113_130522
2019-11-13 13:05:22,702 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 13:05:22,704 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=740, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54867, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 13:05:22,704 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000202C11A9348>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000202C11A9348> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 13:05:22,709 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000202C11A9348> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:05:22,709 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000202C11A9348> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:05:22,710 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x00000202C11A9348> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:05:22,710 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 13:05:22,715 - root - INFO - [x] Sent {"config.ini": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\config.ini", "File Name": "config.ini", "File Size": 245, "MD5": "f206d6f92489b0bfb7116b1f9537c14c", "SHA1": "cc36c5ab75685e7f09eb5a80a1592efb106d1b3b"}]} to queue!
2019-11-13 13:05:22,716 - root - INFO - [x] Sent {"create_db_table.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\create_db_table.py", "File Name": "create_db_table.py", "File Size": 2180, "MD5": "85242881acb57bf4586b92b25a33ce87", "SHA1": "fa29674302cdafa0b2dfdcc1065706ab00714770"}]} to queue!
2019-11-13 13:05:22,717 - root - INFO - [x] Sent {"listdir.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir.py", "File Name": "listdir.py", "File Size": 9516, "MD5": "f344486193d2d0dab11b12dbf2eee4b7", "SHA1": "d7e9373d77c8f23ce533f63febab76fa15a5d3d6"}]} to queue!
2019-11-13 13:05:22,719 - root - INFO - [x] Sent {"listdir_error.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_error.log", "File Name": "listdir_error.log", "File Size": 3581, "MD5": "865a8e5ecbdd544388eb606891ac7fee", "SHA1": "89c685ce17bebad2cfb675e84331febdad89d0c8"}]} to queue!
2019-11-13 13:05:22,725 - root - INFO - [x] Sent {"listdir_info.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_info.log", "File Name": "listdir_info.log", "File Size": 44293, "MD5": "ea84bc9d3228f8d80119e01ce4116053", "SHA1": "ca85190e40eaec44b454b0d79ef6d9e9df377f2f"}]} to queue!
2019-11-13 13:05:22,726 - root - INFO - [x] Sent {"logging_listdir.yaml": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\logging_listdir.yaml", "File Name": "logging_listdir.yaml", "File Size": 861, "MD5": "7ee106c8c2dc81c0cd98ad21d4b2481f", "SHA1": "984571cf9fb34a7c9747a9d2dbd36bd0bbb7549c"}]} to queue!
2019-11-13 13:05:22,727 - root - INFO - [x] Sent {"MANIFEST.in": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\MANIFEST.in", "File Name": "MANIFEST.in", "File Size": 26, "MD5": "0d46f74b56ba4d6c16fe2ac11d437a32", "SHA1": "e0613788cd5822eb7a958bb0ffdfed522c9ef43b"}]} to queue!
2019-11-13 13:05:22,728 - root - INFO - [x] Sent {"sample.csv.csv": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\sample.csv.csv", "File Name": "sample.csv.csv", "File Size": 3922, "MD5": "3cc623eed556fd4dd8c3922b209f89c9", "SHA1": "ebc128aa0b2ef88228711b9bb2a42bdf76d60071"}]} to queue!
2019-11-13 13:05:22,729 - root - INFO - [x] Sent {"test.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\test.py", "File Name": "test.py", "File Size": 608, "MD5": "f0e8fe7e46fb2b4ec0a571cb689f2437", "SHA1": "21fc9ad28a03f8636c37e7a818bb62f88230092a"}]} to queue!
2019-11-13 13:05:22,730 - root - INFO - [x] Sent {"__init__.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__init__.py", "File Name": "__init__.py", "File Size": 280, "MD5": "6036a582623e1da0890729d34c84874b", "SHA1": "92f8a64d57c926c6d37e487f5ea6a2d979b03f7d"}]} to queue!
2019-11-13 13:05:22,732 - root - INFO - [x] Sent {"create_db_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_db_table.cpython-37.pyc", "File Name": "create_db_table.cpython-37.pyc", "File Size": 2196, "MD5": "4b5c10918601efdbf85c9eaf4a3fb9a8", "SHA1": "801d646d90f2795cf47db3ede1b718b5f256aa0d"}]} to queue!
2019-11-13 13:05:22,736 - root - INFO - [x] Sent {"create_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_table.cpython-37.pyc", "File Name": "create_table.cpython-37.pyc", "File Size": 939, "MD5": "0f0659634a6217b8273fd46dd2c22e24", "SHA1": "e2eb26a536a703736c6236fab21de58c09250106"}]} to queue!
2019-11-13 13:05:22,738 - root - INFO - [x] Sent {"listdir.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\listdir.cpython-37.pyc", "File Name": "listdir.cpython-37.pyc", "File Size": 7686, "MD5": "90d04534725a015670e534329df7f43a", "SHA1": "a28e576d8e331787dc6644dc3e6dd780051282a2"}]} to queue!
2019-11-13 13:05:22,741 - root - INFO - [x] Sent {"__init__.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\__init__.cpython-37.pyc", "File Name": "__init__.cpython-37.pyc", "File Size": 366, "MD5": "9b2bc2af4c7f78c59c397c0040006c82", "SHA1": "f026e605e13aa3eb3d17edeae9bfee768bacb081"}]} to queue!
2019-11-13 13:09:52,632 - __main__ - INFO - Naming default_file
2019-11-13 13:09:52,633 - __main__ - INFO - Result: default_file_20191113_130952
2019-11-13 13:09:56,687 - __main__ - INFO - Naming default_file
2019-11-13 13:09:56,687 - __main__ - INFO - Result: default_file_20191113_130956
2019-11-13 13:09:56,696 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 13:09:56,697 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=752, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54916, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 13:09:56,698 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000029322188508>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000029322188508> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 13:09:56,702 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000029322188508> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:09:56,703 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000029322188508> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:09:56,703 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000029322188508> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:09:56,704 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 13:09:56,706 - root - INFO - [x] Sent {"config.ini": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\config.ini", "File Name": "config.ini", "File Size": 245, "MD5": "f206d6f92489b0bfb7116b1f9537c14c", "SHA1": "cc36c5ab75685e7f09eb5a80a1592efb106d1b3b"}]} to queue!
2019-11-13 13:09:56,707 - root - INFO - [x] Sent {"create_db_table.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\create_db_table.py", "File Name": "create_db_table.py", "File Size": 2180, "MD5": "85242881acb57bf4586b92b25a33ce87", "SHA1": "fa29674302cdafa0b2dfdcc1065706ab00714770"}]} to queue!
2019-11-13 13:09:56,708 - root - INFO - [x] Sent {"listdir.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir.py", "File Name": "listdir.py", "File Size": 9516, "MD5": "f344486193d2d0dab11b12dbf2eee4b7", "SHA1": "d7e9373d77c8f23ce533f63febab76fa15a5d3d6"}]} to queue!
2019-11-13 13:09:56,709 - root - INFO - [x] Sent {"listdir_error.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_error.log", "File Name": "listdir_error.log", "File Size": 3581, "MD5": "865a8e5ecbdd544388eb606891ac7fee", "SHA1": "89c685ce17bebad2cfb675e84331febdad89d0c8"}]} to queue!
2019-11-13 13:09:56,718 - root - INFO - [x] Sent {"listdir_info.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_info.log", "File Name": "listdir_info.log", "File Size": 51043, "MD5": "05c26aa59539ebf87fdfec88e4bec130", "SHA1": "1912c4775a06c529c1694bc20443e4426bcc6c47"}]} to queue!
2019-11-13 13:09:56,719 - root - INFO - [x] Sent {"logging_listdir.yaml": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\logging_listdir.yaml", "File Name": "logging_listdir.yaml", "File Size": 861, "MD5": "7ee106c8c2dc81c0cd98ad21d4b2481f", "SHA1": "984571cf9fb34a7c9747a9d2dbd36bd0bbb7549c"}]} to queue!
2019-11-13 13:09:56,720 - root - INFO - [x] Sent {"MANIFEST.in": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\MANIFEST.in", "File Name": "MANIFEST.in", "File Size": 26, "MD5": "0d46f74b56ba4d6c16fe2ac11d437a32", "SHA1": "e0613788cd5822eb7a958bb0ffdfed522c9ef43b"}]} to queue!
2019-11-13 13:09:56,721 - root - INFO - [x] Sent {"sample.csv.csv": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\sample.csv.csv", "File Name": "sample.csv.csv", "File Size": 3922, "MD5": "3cc623eed556fd4dd8c3922b209f89c9", "SHA1": "ebc128aa0b2ef88228711b9bb2a42bdf76d60071"}]} to queue!
2019-11-13 13:09:56,722 - root - INFO - [x] Sent {"test.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\test.py", "File Name": "test.py", "File Size": 608, "MD5": "f0e8fe7e46fb2b4ec0a571cb689f2437", "SHA1": "21fc9ad28a03f8636c37e7a818bb62f88230092a"}]} to queue!
2019-11-13 13:09:56,723 - root - INFO - [x] Sent {"__init__.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__init__.py", "File Name": "__init__.py", "File Size": 280, "MD5": "6036a582623e1da0890729d34c84874b", "SHA1": "92f8a64d57c926c6d37e487f5ea6a2d979b03f7d"}]} to queue!
2019-11-13 13:09:56,725 - root - INFO - [x] Sent {"create_db_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_db_table.cpython-37.pyc", "File Name": "create_db_table.cpython-37.pyc", "File Size": 2196, "MD5": "4b5c10918601efdbf85c9eaf4a3fb9a8", "SHA1": "801d646d90f2795cf47db3ede1b718b5f256aa0d"}]} to queue!
2019-11-13 13:09:56,727 - root - INFO - [x] Sent {"create_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_table.cpython-37.pyc", "File Name": "create_table.cpython-37.pyc", "File Size": 939, "MD5": "0f0659634a6217b8273fd46dd2c22e24", "SHA1": "e2eb26a536a703736c6236fab21de58c09250106"}]} to queue!
2019-11-13 13:09:56,729 - root - INFO - [x] Sent {"listdir.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\listdir.cpython-37.pyc", "File Name": "listdir.cpython-37.pyc", "File Size": 7686, "MD5": "90d04534725a015670e534329df7f43a", "SHA1": "a28e576d8e331787dc6644dc3e6dd780051282a2"}]} to queue!
2019-11-13 13:09:56,732 - root - INFO - [x] Sent {"__init__.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\__init__.cpython-37.pyc", "File Name": "__init__.cpython-37.pyc", "File Size": 366, "MD5": "9b2bc2af4c7f78c59c397c0040006c82", "SHA1": "f026e605e13aa3eb3d17edeae9bfee768bacb081"}]} to queue!
2019-11-13 13:11:13,247 - __main__ - INFO - Naming default_file
2019-11-13 13:11:13,247 - __main__ - INFO - Result: default_file_20191113_131113
2019-11-13 13:11:13,256 - pika.adapters.utils.connection_workflow - INFO - Pika version 1.1.0 connecting to ('::1', 5672, 0, 0)
2019-11-13 13:11:13,256 - pika.adapters.utils.io_services_utils - INFO - Socket connected: <socket.socket fd=784, family=AddressFamily.AF_INET6, type=SocketKind.SOCK_STREAM, proto=6, laddr=('::1', 54929, 0, 0), raddr=('::1', 5672, 0, 0)>
2019-11-13 13:11:13,257 - pika.adapters.utils.connection_workflow - INFO - Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000026EDB2291C8>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000026EDB2291C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2019-11-13 13:11:13,261 - pika.adapters.utils.connection_workflow - INFO - AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000026EDB2291C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:11:13,262 - pika.adapters.utils.connection_workflow - INFO - AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000026EDB2291C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:11:13,262 - pika.adapters.blocking_connection - INFO - Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x0000026EDB2291C8> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2019-11-13 13:11:13,263 - pika.adapters.blocking_connection - INFO - Created channel=1
2019-11-13 13:11:13,265 - root - INFO - [x] Sent {"config.ini": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\config.ini", "File Name": "config.ini", "File Size": 245, "MD5": "f206d6f92489b0bfb7116b1f9537c14c", "SHA1": "cc36c5ab75685e7f09eb5a80a1592efb106d1b3b"}]} to queue!
2019-11-13 13:11:13,266 - root - INFO - [x] Sent {"create_db_table.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\create_db_table.py", "File Name": "create_db_table.py", "File Size": 2180, "MD5": "85242881acb57bf4586b92b25a33ce87", "SHA1": "fa29674302cdafa0b2dfdcc1065706ab00714770"}]} to queue!
2019-11-13 13:11:13,267 - root - INFO - [x] Sent {"listdir.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir.py", "File Name": "listdir.py", "File Size": 9516, "MD5": "f344486193d2d0dab11b12dbf2eee4b7", "SHA1": "d7e9373d77c8f23ce533f63febab76fa15a5d3d6"}]} to queue!
2019-11-13 13:11:13,268 - root - INFO - [x] Sent {"listdir_error.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_error.log", "File Name": "listdir_error.log", "File Size": 3581, "MD5": "865a8e5ecbdd544388eb606891ac7fee", "SHA1": "89c685ce17bebad2cfb675e84331febdad89d0c8"}]} to queue!
2019-11-13 13:11:13,286 - root - INFO - [x] Sent {"listdir_info.log": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\listdir_info.log", "File Name": "listdir_info.log", "File Size": 57646, "MD5": "dca92cf4c9afac2804190f892a5815bd", "SHA1": "9fb87f75d2f526b725d0955ff5630dfe9b8b632c"}]} to queue!
2019-11-13 13:11:13,287 - root - INFO - [x] Sent {"logging_listdir.yaml": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\logging_listdir.yaml", "File Name": "logging_listdir.yaml", "File Size": 861, "MD5": "7ee106c8c2dc81c0cd98ad21d4b2481f", "SHA1": "984571cf9fb34a7c9747a9d2dbd36bd0bbb7549c"}]} to queue!
2019-11-13 13:11:13,288 - root - INFO - [x] Sent {"MANIFEST.in": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\MANIFEST.in", "File Name": "MANIFEST.in", "File Size": 26, "MD5": "0d46f74b56ba4d6c16fe2ac11d437a32", "SHA1": "e0613788cd5822eb7a958bb0ffdfed522c9ef43b"}]} to queue!
2019-11-13 13:11:13,289 - root - INFO - [x] Sent {"sample.csv.csv": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\sample.csv.csv", "File Name": "sample.csv.csv", "File Size": 3922, "MD5": "3cc623eed556fd4dd8c3922b209f89c9", "SHA1": "ebc128aa0b2ef88228711b9bb2a42bdf76d60071"}]} to queue!
2019-11-13 13:11:13,290 - root - INFO - [x] Sent {"test.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\test.py", "File Name": "test.py", "File Size": 608, "MD5": "f0e8fe7e46fb2b4ec0a571cb689f2437", "SHA1": "21fc9ad28a03f8636c37e7a818bb62f88230092a"}]} to queue!
2019-11-13 13:11:13,291 - root - INFO - [x] Sent {"__init__.py": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__init__.py", "File Name": "__init__.py", "File Size": 280, "MD5": "6036a582623e1da0890729d34c84874b", "SHA1": "92f8a64d57c926c6d37e487f5ea6a2d979b03f7d"}]} to queue!
2019-11-13 13:11:13,293 - root - INFO - [x] Sent {"create_db_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_db_table.cpython-37.pyc", "File Name": "create_db_table.cpython-37.pyc", "File Size": 2196, "MD5": "4b5c10918601efdbf85c9eaf4a3fb9a8", "SHA1": "801d646d90f2795cf47db3ede1b718b5f256aa0d"}]} to queue!
2019-11-13 13:11:13,295 - root - INFO - [x] Sent {"create_table.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\create_table.cpython-37.pyc", "File Name": "create_table.cpython-37.pyc", "File Size": 939, "MD5": "0f0659634a6217b8273fd46dd2c22e24", "SHA1": "e2eb26a536a703736c6236fab21de58c09250106"}]} to queue!
2019-11-13 13:11:13,297 - root - INFO - [x] Sent {"listdir.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\listdir.cpython-37.pyc", "File Name": "listdir.cpython-37.pyc", "File Size": 7686, "MD5": "90d04534725a015670e534329df7f43a", "SHA1": "a28e576d8e331787dc6644dc3e6dd780051282a2"}]} to queue!
2019-11-13 13:11:13,298 - root - INFO - [x] Sent {"__init__.cpython-37.pyc": [{"Parent Path": "C:\\Users\\TEU_USER\\PycharmProjects\\listdir\\listdir\\__pycache__\\__init__.cpython-37.pyc", "File Name": "__init__.cpython-37.pyc", "File Size": 366, "MD5": "9b2bc2af4c7f78c59c397c0040006c82", "SHA1": "f026e605e13aa3eb3d17edeae9bfee768bacb081"}]} to queue!
